.eslintignore

```diff
@@ -4,4 +4,6 @@
 /examples
 /mithril.js
 /mithril.min.js
+/stream.js
```

Comment:
Nit: use `mithril-stream.js` and `mithril-stream.min.js` instead, so it doesn't override `mithril/stream`.